components/src/core/Drawer/DrawerRailItem/DrawerRailItem.tsx

```diff
@@ -42,48 +42,49 @@ const useUtilityClasses = (ownerState: DrawerRailItemProps): Record<DrawerRailIt
 };
 
 export type ExtendedNavItem = NavItem & { ButtonBaseProps?: Partial<MuiButtonBaseProps> };
-export type DrawerRailItemProps = SharedStyleProps & {
-    /** Custom classes for default style overrides */
-    classes?: DrawerRailItemClasses;
+export type DrawerRailItemProps = SharedStyleProps &
+    MuiButtonBaseProps & {
```

Comment:
This is a little weird. Should we be deprecating the ButtonBaseProps prop below? How is that currently being used?

Reply:
Currently, it is not accepting style in DrawerNavItem

Reply:
@joebochill @bkarambe did we ever clear this up? What's the approach we want to take here?

Reply:
Yes, we will be using MuiButtonBaseProps in DrawerRailItemProps